amundsen_application/static/js/components/DashboardPage/ShimmeringDashboardLoader/styles.scss

```diff
@@ -1,26 +1,33 @@
-@import 'variables';
+@import "variables";
 
 $loading-duration: 1s;
```

Comment:
Just took a look on staging, the animation seems somewhat fast. This is a subjective feeling. Maybe 2 or 3s would be better?

Reply:
I agree with you. I liked the 2s way more.

We are copying the styling from the LPL version, and it is a 1s there (almost hysterical IMO)

Reply:
I don't think we need to follow LPL 100%. I'll leave you with the final call on `$loading-duration`.

Reply:
We could raise this to Anita. There's no official mandate that we have to follow LPL Data, but consistency across our tools is the longterm goal for Data Products so it could hurt us down the line to pick and choose certain specifications based on what we happen to like right now.  Anita is the designer thinking about potential policies around official open source design, so when the time comes I encourage us to engage with her on thoughts on what policies work best for us. Some ideas I can think of are exceptions for open source contributions where we don't want to block the feature on design, and of course technical limitations like how we're unable to provide the ideal search filter UI until certain search service features are available.

Reply:
Makes sense!
We could raise it on this week's Design Sync on Thursday

Reply:
The sync for this Thursday is likely just with Wiebke and Knowl for updates on features, but Anita should still be hosting office hours. I'll send you info on slack.